app/src/common/shared/com/igalia/wolvic/ui/adapters/WebApp.java

```diff
@@ -87,38 +67,53 @@ public String getId() {
     }
 
     public String getName() {
-        return mName;
+        return mManifest.getName();
     }
 
     public String getShortName() {
-        return mShortName;
+        return mManifest.getShortName();
     }
 
     public String getScope() {
-        return mScope;
+        return mManifest.getScope();
     }
 
     public String getStartUrl() {
-        return mStartUrl;
+        return mManifest.getStartUrl();
     }
 
     public int getThemeColor() {
-        return mThemeColor;
+        if (mManifest.getThemeColor() != null) {
+            return mManifest.getThemeColor();
+        } else {
+            return Color.TRANSPARENT;
+        }
```

Comment:
I like the ternary operator for these cases
```
return mManifest.getThemeColor() != null ? mManifest.getThemeColor() : Color.TRANSPARENT
```
you can also cache the result of `getThemeColor()` although it isn't a big deal

Reply:
Done.